sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -293,10 +293,9 @@ await foreach (BasicDigitalTwin twin in asyncPageableResponse)
                 // This code snippet demonstrates querying digital twin results using an AdtQueryBuilder, an object that allows for 
                 // fluent-style query construction that makes it easier to write queries.
                 AsyncPageable<BasicDigitalTwin> asyncPageableResponseQueryBuilder = client.QueryAsync<BasicDigitalTwin>(
-                    new AdtQueryBuilder()
+                    new DigitalTwinsQueryBuilder()
```

Comment:
I am not sure, but should we be calling Build here?

Reply:
Yes we should, great catch